Mathlib/NumberTheory/Padics/PadicVal.lean

```diff
@@ -549,6 +557,12 @@ theorem padicValNat_factorial_mul {p : ℕ} (n : ℕ) (hp : p.Prime):
       padicValNat_def' (Nat.Prime.ne_one hp) <| factorial_pos n]
   exact Prime.multiplicity_factorial_mul hp
 
+theorem multiplicity_choose {p n k b : ℕ} [hp : Fact p.Prime] (hkn : k ≤ n) (hnb : log p n < b) :
+    padicValNat p (choose n k) =
+    ((Finset.Ico 1 b).filter fun i => p ^ i ≤ k % p ^ i + (n - k) % p ^ i).card :=
```

Comment:
Shouldn't this be:
```suggestion
theorem padicValNat_choose {p n k : ℕ} [hp : Fact p.Prime] (hkn : k ≤ n) :
    padicValNat p (choose n k) =
    ((Finset.Icc 1 (log p n)).filter fun i ↦ p ^ i ≤ k % p ^ i + (n - k) % p ^ i).card :=
```

Reply:
Yes, at least the name should have been `padicValNat_choose`. But I think it would be good to keep the additional hypothesis `hnb` and the `b`, because
 - it it slighltly stronger to allow adding more zeros here
 - it is done this way in I think it might be good to keep https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Nat/Multiplicity.html#Nat.Prime.multiplicity_choose
and

Reply:
OK I agree, with you now.